REORG_LOSSES.md

```diff
@@ -0,0 +1,19 @@
+# Reorg losses
```

Comment:
Could you create a `/docs/` folder and add this there? (imo ideally with a lowercase filename)

I think it's better to start the directory because over time we'll accumulate more docs and it's better to have them together in a single place, instead of scattered on the project root. Alternatively, the wiki might also be a good place, but that has the downside of then not being self-contained in a clone of the repo.